### ISSUE TITLE
[enhancement]: Batch Size on the web gui

### ISSUE THREAD

**Issue by AshtakaOOf** [invoke-ai/InvokeAI#1562]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

discord: Ashtaka#0043

### What should this feature add?

Add batch size to the web gui instead of only having only the amount of batch to make

### Alternatives

idk...

### Aditional Content

![image](https://user-images.githubusercontent.com/66513643/204035677-6e25a6b2-0ce6-48fc-8e40-19f2f07fd8c7.png)
![image](https://user-images.githubusercontent.com/66513643/204035704-f0a1c41b-03c8-45bc-98ce-94ce997eae6b.png)

**Comment by davr** [comment]
Making multiple images in parallel in a batch significantly speeds up image generation in my experience if you have the VRAM to spare

**Comment by fish-face** [comment]
Better than this, work out how many images can be generated in one batch without running out of RAM, and run batches automatically.

**Comment by zeddyemmy** [comment]
i cant tell if this is hidden somewhere in the interface, but just tried it out for the new v3 - this is unfortunately a deciding factor that would prevent me from considering using InvokeAI, as I very often make large numbers of images and iterate upon that, and the performance is paramount

**Comment by AshtakaOOf** [comment]
From what I know from Discord, the devs told me that the backend is very different from other webuis.
They switched to Diffusers since then so maybe it could be considered again, so don't hesitate to go and discuss it on discord with the devs.

**Comment by radry** [comment]
Multi size batches should work on diffusers as well. SD.next (A1111 fork) also uses diffusers and can use batch size > 1

**Comment by rubymignot** [comment]
I would really love that yeah

**Comment by DustyShoe** [comment]
I’m closing this feature request as not planned.

InvokeAI is primarily designed for iterative and interactive generation workflows, not for high-throughput batch image generation. As such, adding batch size controls to the web UI does not align well with the project’s core goals and usage patterns.

**Comment by AshtakaOOf** [comment]
> I’m closing this feature request as not planned.
> 
> InvokeAI is primarily designed for iterative and interactive generation workflows, not for high-throughput batch image generation. As such, adding batch size controls to the web UI does not align well with the project’s core goals and usage patterns.

I think this feature would still be worthwhile for slower gpus to do in/outpainting on the canvas.

**Comment by DustyShoe** [comment]
This feature request has been open for four years without implementation, which suggests it has not been considered a priority. I share that assessment.

**Comment by AshtakaOOf** [comment]
I concur on that part, it is indeed not a priority.
